src/lib/event_mapper/impl/glue_crawler_event_mapper.py

```diff
@@ -0,0 +1,36 @@
+from .general_aws_event_mapper import GeneralAwsEventMapper
+from ...settings import Settings
+from ...core.constants import EventSeverity
+
+
+class GlueCrawlerEventMapper(GeneralAwsEventMapper):
+    def __init__(self, settings: Settings):
+        super().__init__(settings)
+
+    def get_resource_name(self, event):
+        return event["detail"]["crawlerName"]
+
+    def get_resource_state(self, event):
+        return event["detail"]["state"]
+
+    def get_event_severity(self, event):
```

Comment:
as per sync-up discussion, we might need event_result rather than event_severity

Reply:
Already fixed